console/templates/store.html

```diff
@@ -8,88 +8,95 @@
 
-          <label class="col-md-1 control-label" for="survey_id">survey_id:</label> <br>
-          <input type="number" class="form-control" id="survey_id" name="survey_id"> <br> <br>
+            <label class="col-md-1 control-label" for="survey_id">survey_id:</label> <br>
+            <input type="number" class="form-control" id="survey_id" name="survey_id"> <br> <br>
 
-          <label class="col-md-1 control-label" for="datetime_earliest">datetime_earliest: </label>
-          <input type="datetime-local" class="form-control" id="datetime_earliest" name="datetime_earliest"> <br>
+            <label class="col-md-1 control-label" for="datetime_earliest">datetime_earliest: </label>
+            <input type="datetime-local" class="form-control" id="datetime_earliest" name="datetime_earliest"> <br>
 
-          <label class="col-md-1 control-label" for="datetime_latest">datetime_latest: </label>
-          <input type="datetime-local" class="form-control" id="datetime_latest" name="datetime_latest"> <br> <br>
+            <label class="col-md-1 control-label" for="datetime_latest">datetime_latest: </label>
+            <input type="datetime-local" class="form-control" id="datetime_latest" name="datetime_latest"> <br> <br>
 
+          </div>
+
+          <span class="input-group-btn">
+              <button class="btn btn-default" type="submit">Search</button>
+          </span>
+          {% if form.errors %}
+              <ul class=errors>
+              {% for error in form.errors %}
+                <li>{{error}} : {{ form.errors[error][0] }}</li>
+              {% endfor %}
+              </ul>
+          {% endif %}
         </div>
-        <span class="input-group-btn">
-            <button class="btn btn-default" type="submit">Search</button>
-        </span>
-        {% if form.errors %}
-            <ul class=errors>
-            {% for error in form.errors %}
-              <li>{{error}} : {{ form.errors[error][0] }}</li>
-            {% endfor %}
-            </ul>
-        {% endif %}
       </div>
-    </div>
-  </form>
+    </form>
+
+    <div class="table">
+      <table class="table" width="100%">
+        <thead class="table__head table__header-cell--align">
+          <tr>
+            <th class="table__header-cell table__header-cell--align"> tx_id </th>
+            <th class="table__header-cell"> survey_id </th>
+            <th class="table__header-cell"> ru_ref </th>
+            <th class="table__header-cell"> user_id </th>
+            <th class="table__header-cell"> submitted_at </th>
+            <th class="table__header-cell"> invalid </th>
+            <th class="table__header-cell"> Reprocess </th>
+          </tr>
+        </thead>
+        <tbody>
+          <form id="reprocess-form" name='reprocess-form' class="form-horizontal" method='POST' action="/reprocess">
+            {% if request.path == '/store' %}
+            {% else %}
```

Comment:
This is how I've changed it so that it doesn't load all the data first, suggestions on a better way would be appreciated 👍